protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -644,6 +644,13 @@ func genStringValidator(g *protogen.GeneratedFile, f *protogen.Field, varName st
 		}
 	}
 
+	if rules.GetPrefix() != "" {
+		prefix := rules.GetPrefix()
+		g.P("if !", stringsPackage.Ident("HasPrefix"), "(", varName, ", \"", prefix, "\") {")
+		genErrorString(g, varName, string(f.Desc.Name()), "start with prefix: "+prefix)
```

Comment:
You're giving away the prefix here. Not a big concern, but it might be better to make the message generic.